File: OCP/ Kata_ Shops Types/SolucionJava/Shop/Shop/Type/SuperMarket.java
package SolucionJava.Shop.Shop.Type;

public class SuperMarket implements FoodShopInterface {
    private final Boolean SELL_PRECOOK = false;
    private final String HUMAN_TYPE_NAME = "SuperMarket";

    public String getType() {
        return HUMAN_TYPE_NAME;
    }


    public Boolean sellPreCook() {
        return SELL_PRECOOK;
    }
}

File: OCP/ Kata_ Shops Types/SolucionPhp/Shop/Shop/Type/SuperMarket.php
<?php

declare(strict_types=1);

namespace SolucionPhp\Shop\Shop\Type;

class SuperMarket implements FoodShopInterface
{
    private const HUMAN_TYPE = 'SuperMarket';
    private const SELL_PRECOOK = false;

    public function getType(): string
    {
        return self::HUMAN_TYPE;
    }

    public function sellPreCook(): bool
    {
        return self::SELL_PRECOOK;
    }

}
